pkg/backend/refresh.go

```diff
@@ -2,11 +2,29 @@ package backend
 
 import (
 	"context"
+	"time"
 
+	"github.com/hashicorp/vault/sdk/framework"
 	"github.com/hashicorp/vault/sdk/logical"
 	"golang.org/x/oauth2"
 )
 
+func tokenOk2Reuse(tok *oauth2.Token, data *framework.FieldData) bool {
+	if !tok.Valid() {
+		return false
+	}
+	if data == nil {
+		return true
+	}
+	if minsecondsstr, ok := data.GetOk("minimum_seconds"); ok {
+		minseconds := minsecondsstr.(int)
```

Comment:
I believe you need to use `strconv.ParseInt` here.

Reply:
This way of conversion works too.  All the other examples I found use a ".(string)" there, so why not use ".(int)"?

Reply:
Got it! Sorry, I forgot this stuff was properly defined by a schema. This should be fine.